### PR TITLE
removed 1920_1080 support, I think....

### PR DESCRIPTION
**Pull request by Master-Matt**
Tested with my hdin. ran all 3 fine. Will test with sorc to be sure.

### CODE REVIEW

File: src/pather.py [hunk 1]
```diff
@@ -49,46 +49,46 @@ def __init__(self, screen: Screen, template_finder: TemplateFinder):
...
-            5: {"A5_TOWN_2": (555-100, 429-100), "A5_TOWN_3": (-285-100, 51-100), "A5_TOWN_4": (358-100, 15-100)},
-            6: {"A5_TOWN_3": (-775-100, 382-120), "A5_TOWN_4": (-132-100, 346-120), "A5_TOWN_5": (80-100, -240-120), "A5_TOWN_6": (560-100, 211-120)},
-            8: {"A5_TOWN_6": (190, 440), "A5_TOWN_5": (-323+30, 192-200), "A5_TOWN_7": (867+30, 69-200)},
-            9: {"A5_TOWN_5": (-611, 250), "A5_TOWN_7": (579, 127)},
-            10: {"A5_TOWN_4": (-708, 87), "A5_TOWN_6": (-16, -48), "A5_TOWN_8": (482, 196)},
-            11: {"A5_TOWN_6": (-448, -322), "A5_TOWN_8": (50, -78), "A5_TOWN_9": (11, 346)},
-            12: {"A5_TOWN_8": (-209, -294), "A5_TOWN_9": (-248, 130)},
-            13: {"A5_TOWN_3": (180, 180),"A5_TOWN_10": (-800, 332), "A5_TOWN_4": (822, 146)},
-            14: {"A5_TOWN_3": (670, 260), "A5_TOWN_10": (-300, 420)},
+            0: {'A5_TOWN_0': (19, 182), 'A5_TOWN_1': (-67, -100)},
```
Comment (aeon0) [inline src/pather.py:56]: how did you convert all these coordinates? Cause I dont think the coordinates are correct. Example:
(110-70)/1.5 = 26.6
373/1.5 = 248.6

File: src/char/i_char.py [hunk 2]
```diff
@@ -129,7 +129,7 @@ def tp_town(self):
             )
             if success1 or success2:
                 pos = pos1 if success1 else pos2
-                pos = (pos[0], pos[1] + (45 * self._config.scale))
+                pos = (pos[0], pos[1] + 45 )
```
Comment (aeon0) [inline src/char/i_char.py:132]: That would be + 30 I think

File: game.ini [hunk 3]
```diff
@@ -16,110 +16,113 @@ health_potion_1=0,100,76,10,255,255
...
-eldritch_end=1012,42
-shenk_end=1600,816
+pindle_save_dist=921,35, 1123,70, 960,88
+eldritch_save_dist=652,63, 563,73
+pindle_end=1037,174
+eldritch_end=675,28
+shenk_end=1067,544
+eldritch_save_tp=640,533
+pindle_save_tp=400,27
+shenk_save_tp=437,215
```
Comment (aeon0) [inline game.ini:128]: The save_tp positions are not used anymore, can be deleted

**Review by aeon0 — CHANGES_REQUESTED** [review]
Thx for the help!

File: game.ini (hunk 3 above)
Comment (Master-Matt) [reply game.ini]: done

**Comment by Master-Matt** [comment]
Should be good now

**Review by aeon0 — APPROVED** [review]

**Comment by aeon0** [comment]
So I put a lot of effort into supporting multiple res and you just want to delete it???!?

**Comment by aeon0** [comment]
Merged :) THX

**Pull request merged**